modules/rosa-hcp-sts-creating-a-cluster-cli-no-cni-plugin.adoc

```diff
@@ -69,16 +74,14 @@ The following `State` field changes are listed in the output as the cluster inst
 ====
 If the installation fails or the `State` field does not change to `ready` after more than 10 minutes, check the installation troubleshooting documentation for details. For more information, see _Troubleshooting installations_. For steps to contact Red Hat Support for assistance, see _Getting support for Red Hat OpenShift Service on AWS_.
 ====
-+
-[IMPORTANT]
-====
-When you first log in to the cluster after it reaches `ready` status, the nodes will still be in the `not ready` state until you install your own CNI plugin. After CNI installation, the nodes will change to `ready`. 
-====
 
 . Track the progress of the cluster creation by watching the {product-title} installation program logs. To check the logs, run the following command:
 +
 [source,terminal]
 ----
 $ rosa logs install --cluster=<cluster_name> --watch <1>
 ----
-<1> Optional: To watch for new log messages as the installation progresses, use the `--watch` argument.
\ No newline at end of file
+<1> Optional: To watch for new log messages as the installation progresses, use the `--watch` argument.
+
+== Expected behavior for clusters without a CNI plugin
+Although {hcp-title} cluster installation is complete, the cluster cannot operate without a CNI plugin. Because the nodes are not ready, the workloads cannot deploy. For example, the OpenShift cluster console is not available, so you must use OpenShift CLI (OC) to log in to the cluster. Additionally, other OpenShift components such as the HA-Proxy-based Ingress Controller, image registry, and prometheus-based monitoring stack are not running. This is _expected behavior_ until you install a CNI provider.
```

Comment:
* Use `{product-title}`. Also I feel like "[web console](https://82773--ocpdocs-pr.netlify.app/openshift-rosa/latest/web_console/web-console-overview)" is more common.
* Use `{oc-first}`.
* Not sure about my "HAProxy-based Ingress Controller" spelling suggestion, but that's how I see it in [a few spots](https://docs.openshift.com/container-platform/4.17/machine_management/creating-infrastructure-machinesets.html#infrastructure-components_creating-infrastructure-machinesets) around the repo.
* [Repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) technically do allow underscores for emphasis, but typically should be saved for first instances of a new term. Not sure the emphasis is super critical here.
```suggestion
Although {hcp-title} cluster installation is complete, the cluster cannot operate without a CNI plugin. Because the nodes are not ready, the workloads cannot deploy. For example, the {product-title} cluster web console is not available, so you must use the {oc-first} to log in to the cluster. Additionally, other OpenShift components such as the HAProxy-based Ingress Controller, image registry, and prometheus-based monitoring stack are not running. This is expected behavior until you install a CNI provider.
```
Also maybe break up the paragraph, if you like:

```suggestion
Although {hcp-title} cluster installation is complete, the cluster cannot operate without a CNI plugin. Because the nodes are not ready, the workloads cannot deploy. For example, the {product-title} cluster web console is not available, so you must use the {oc-first} to log in to the cluster.

Additionally, other OpenShift components such as the HAProxy-based Ingress Controller, image registry, and prometheus-based monitoring stack are not running. This is expected behavior until you install a CNI provider.
```